flask_appbuilder/security/manager.py

```diff
@@ -261,11 +263,27 @@ def __init__(self, appbuilder):
                 self.oauth_remotes[provider_name] = obj_provider
 
         # Setup Flask-Login
+        self.set_login_manager(app)
+
+        # Setup Flask-Jwt-Extended
+        self.set_jwt_manager(app)
+
+    def set_login_manager(self, app):
+        """
+            Override to implement your custom login manager instance
+
+            :param app: Flask app
+        """
         self.lm = LoginManager(app)
         self.lm.login_view = "login"
         self.lm.user_loader(self.load_user)
 
-        # Setup Flask-Jwt-Extended
+    def set_jwt_manager(self, app):
```

Comment:
Alternate naming / strategy:
```
def build_jwt_manager(self, app): -> JWTManager
```
This allows the `jwt_manager` to remain "private" and still be fully customizable